utils/sub8_ros_tools/sub8_ros_tools/msg_helpers.py

```diff
@@ -113,14 +113,14 @@ def numpy_matrix_to_quaternion(np_matrix):
     return geometry_msgs.Quaternion(*np_quaternion)
 
 
-def numpy_pair_to_pose(np_translation, np_rotation_matrix):
```

Comment:
Switch the depending! To the death!!!

- Jake

> On Apr 30, 2016, at 12:08 AM, mattlangford <notifications@github.com> wrote:
> 
> In utils/sub8_ros_tools/sub8_ros_tools/msg_helpers.py:
> 
> > @@ -113,14 +113,14 @@ def numpy_matrix_to_quaternion(np_matrix):
> >      return geometry_msgs.Quaternion(*np_quaternion)
> >  
> >  
> > -def numpy_pair_to_pose(np_translation, np_rotation_matrix):
> Oh no! Just ran into an issue here. We should probably switch these back or switch the other code that was dependent on the ordering here.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>